From bdc660e666670f87e909656e02a7698886b5d421 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 14 Feb 2023 08:43:37 +0800
Subject: [PATCH] dts: cvc-fl: disable unsupported device nodes

The device nodes of "qspi" and "gmac0" are not supported in linux, so we
need to disable them to eliminate error log while booting.

Upstream-Status: Pending

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi b/arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi
index 03fd154bb8b3..baffa50bb047 100644
--- a/arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32gxxxa-cvc-fl.dtsi
@@ -31,11 +31,13 @@ &usbotg {
 };
 
 &qspi {
-	status = "okay";
+	/* We don't support HyperFlash in linux */
+	status = "disabled";
 };
 
 &gmac0 {
-	status = "okay";
+	/* Before enabling SJA1110, set it to be "disabled" */
+	status = "disabled";
 	phy-mode = "rgmii";
 	phy-handle = <&gmac0_mdio_c_phy19>;
 	pinctrl-names = "default";
-- 
2.36.1

